vector/src/main/java/im/vector/app/features/settings/troubleshoot/TestSystemSettings.kt

```diff
@@ -40,7 +40,6 @@ class TestSystemSettings @Inject constructor(private val context: Context,
             description = stringProvider.getString(R.string.settings_troubleshoot_test_system_settings_failed)
             quickFix = object : TroubleshootQuickFix(R.string.open_settings) {
                 override fun doFix() {
-                    if (manager?.diagStatus == TestStatus.RUNNING) return // wait before all is finished
```

Comment:
ah, in this case the `doFix` is the click listener for the button? (which won't be shown until the correct step)

Reply:
> ah, in this case the doFix is the click listener for the button?

yes